stable/postgresql/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
```

Comment:
How did you arrive at this list and should these names also be added to the maintainer list in the Chart.yaml?

Reply:
This was a test, I'll remove this file